main.go

```diff
@@ -3,7 +3,7 @@ package main
 import (
 	"github.com/hashicorp/terraform/plugin"
 	"github.com/hashicorp/terraform/terraform"
-	"github.com/deviavir/terraform-provider-gsuite/gsuite"
+	"github.com/DeviaVir/terraform-provider-gsuite/gsuite"
```

Comment:
For some reason my machines really do not like this change, breaking on `case-insensitive import collision`

Reply:
Hmm. I can rollback that change.

The impersonated_user_email needs to be an email with admin rights, not the service account email.

Reply:
```
* gsuite_group.test.2: Error creating group: Post https://www.googleapis.com/admin/directory/v1/groups?alt=json: oauth2: cannot fetch token: 401 Unauthorized
Response: {
  "error" : "unauthorized_client",
  "error_description" : "Client is unauthorized to retrieve access tokens using this method."
}
```
Is what I'm getting when I use an admin user (the same one I used previously for the credentials)

Reply:
That's usually an issue with oauth scopes. The service account should be permitted to use all of the oauth scopes defined in gsuite.oauthScopes

I have tried it here, and it works.

Reply:
Worked when using your new `oauth_scopes` 😄

Reply:
Done. Yes please. Add `make test` to travis. I added it to the Makefile.

I needed to fix some fmt errors for different types.